docs/src/api/class-error.md

```diff
@@ -0,0 +1,26 @@
+# class: Error
```

Comment:
shouldn't it be more specific like PlaywrightError so that it's easier to tell what kind of error is being caught?

Reply:
it should be that would result in a breaking change on Playwright Python side.

Reply:
@yury-s https://github.com/microsoft/playwright-python/issues/653